activiti-cloud-service-common/activiti-cloud-services-common-identity-keycloak/src/main/java/org/activiti/cloud/services/identity/keycloak/KeycloakManagementService.java

```diff
@@ -42,22 +48,31 @@ public KeycloakManagementService(KeycloakClient keycloakClient,
     }
 
     @Override
-    public List<User> findUsers(String key, Set<String> roles, Integer page, Integer size) {
+    public List<User> findUsers(UserSearchParams userSearchParams) {
         return keycloakClient
-            .searchUsers(key, calculateFirst(page, size), size)
+            .searchUsers(userSearchParams.getSearchKey(), PAGE_START, PAGE_SIZE)
```

Comment:
I removed the pagination because we are not using it and because we are applying some manual filters pagination doesn't work properly

Reply:
ok, pagination is not even relevant to the widget that will be using these endpoints on the FE side